docs/src/pages/Conformance.md

```diff
@@ -124,26 +123,52 @@ The whole system history interaction can be used to obtain a list of changes (cr
     curl -k -u '<username>:<password>' 'https://<host>:<port>/fhir-server/api/v4/_history'
 ```
 
-The response is a history bundle as described by the [FHIR specification](https://www.hl7.org/fhir/http.html#history) with one exception. Whereas the specification requires each entry in the history bundle to contain the full contents of the resource (at least for entries with a `entry.request.method` of PUT or POST), the IBM FHIR Server whole-system history response bundle contains only references to the resources. Clients may choose which resources they fetch. They may also fetch the resources in parallel, which may greatly improve throughput.
+The response is a history bundle as described by the [FHIR specification](https://www.hl7.org/fhir/http.html#history) with one exception. Whereas the specification requires each entry in the history bundle to contain the full contents of the resource (at least for entries with a `entry.request.method` of PUT or POST), clients can specify the HTTP header value `Prefer: return=none` in which case the whole-system history response bundle contains only references to the resources. This allows clients to decide which resources they actually need to read.
```

Comment:
should this say `return=minimal` instead of `return=none` or were you trying to introduce a new value?

Reply:
fixed